test/unit/ajax.js

```diff
@@ -1590,6 +1592,19 @@ module( "ajax", {
 				ok( false, "error" );
 			}
 		} );
+
+		if ( jQuery.support.cors ) {
+			ajaxTest( "gh-1684 - jQuery.ajax() - support cross-domain PATCH in IE10-11", 1, {
+				url: "http://httpbin.org/patch",
```

Comment:
>> How would you test jQuery.ajaxSettings.xhr?
>
> In IE env set type to patch and call jQuery.ajaxSettings.xhr then test returned value.

This will only be a duplicate of the implementation from this patch. I don't want to check if I implemented what I think I did, this is pretty trivial. I want to ensure it works.

> In technical terms, testing native API with conjunction of ajax module would be an integration test, testing through stubbing interface would be a unit test.

We already have such tests - PATCH in IE8 is tested in this way.

> In technical terms, testing native API with conjunction of ajax module would be an integration test, testing through stubbing interface would be a unit test.

IMO a test with a stubbed API would be useless here, it would basically just repeat the code I wrote.